examples/lightning_base.py

```diff
@@ -96,25 +93,14 @@ def optimizer_step(self, epoch, batch_idx, optimizer, optimizer_idx, second_orde
         optimizer.zero_grad()
         self.lr_scheduler.step()
 
-    def get_tqdm_dict(self):
-        avg_loss = getattr(self.trainer, "avg_loss", 0.0)
-        tqdm_dict = {"loss": "{:.3f}".format(avg_loss), "lr": self.lr_scheduler.get_last_lr()[-1]}
-        return tqdm_dict
-
-    def test_step(self, batch, batch_nb):
```

Comment:
removing `test_step` and `test_end` will likely break the other lightning examples. Will have to update their code accordingly.